.github/workflows/camunda-platform-release.yml

```diff
@@ -173,15 +173,18 @@ jobs:
           # see https://maven.apache.org/maven-release/maven-release-plugin/perform-mojo.html#workingDirectory
           export ZBCTL_ROOT_DIR=${PWD}
 
-          # -DskipQaBuild=true -P-autoFormat
+          # -DskipQaBuild=true -P-autoFormat,buildDistFrontend,skipFrontendBuild
           #   In order to disable QA/IT and the auto formatting during the release.
           #   It is unnecessary to release integration tests, and we don't want to run the auto-formatting unnecessarily.
+          #   Building Operate and Tasklist frontend packages does not work in concurrent mode:
+          #   - It is necessary to build the frontend packages in a single place, from dist module, using buildDistFrontend profile
+          #   - It is necessary to skip frontend build in the respective Operate and Tasklist client modules using skipFrontendBuild profile
           ./mvnw release:perform -B \
             -Dgpg.passphrase="${{ steps.secrets.outputs.MAVEN_CENTRAL_GPG_SIGNING_KEY_PASSPHRASE }}" \
             -DlocalCheckout=${{ inputs.dryRun }} \
             -DskipQaBuild=true \
-            -P-autoFormat \
-            -Darguments='-P-autoFormat -DskipQaBuild=true -DskipChecks=true -DskipTests=true -Dskip.fe.build=false -Dspotless.apply.skip=false -Dskip.central.release=${SKIP_REPO_DEPLOY} -Dskip.camunda.release=${SKIP_REPO_DEPLOY} -Dzbctl.force -Dzbctl.rootDir=${ZBCTL_ROOT_DIR} -Dgpg.passphrase="${{ steps.secrets.outputs.MAVEN_CENTRAL_GPG_SIGNING_KEY_PASSPHRASE }}"'
```

Comment:
Can't remember anymore. Did we skipped the build per default now and enabled it here again? Do we maybe need to change here as well? Meaning this one `-Dskip.fe.build=false`

Reply:
`-Dskip.fe.build=false` forces the fe build at client modules level, now it is not needed as the fe is built at `dist` level
as I said above, I will simplify this in this pull request as now Tasklist and Operate entrypoints are in `dist` and not in the webapps anymore, stay tuned :)